plugins/git/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-find_package(Git2 REQUIRED)
+find_package(libgit2 REQUIRED)
```

Comment:
I see you have renamed this file to `Findlibgit2.cmake`. If that was to have the original name of the file (I don't know what it was named `FindGit2.cmake`), that should be `FindLibgit2.cmake` and you shall use:

```
find_package(Libgit2 REQUIRED)
```

Reply:
I have renamed it to `FindLibgit2.cmake`, as the `Git2` name could be confusing; and also, all exported symbols are prefixed with *LIBGIT2*.